lab_3_ann_retriever/main.py

```diff
@@ -157,6 +195,25 @@ def build(self) -> bool:
         Returns:
             bool: True if built successfully, False in other case
         """
+        if not self._corpus:
+            return False
+
+        vocab = set()
+        for doc in self._corpus:
+            vocab |= set(doc)
+        self._vocabulary = sorted(list(vocab))
+        if not self._vocabulary:
+            return False
+
+        self._token2ind = {word: index for index, word in enumerate(self._vocabulary)}
+        if self._token2ind is None:
```

Comment:
Strange check: I'm absolutely sure that `self._token2ind` is not None

Reply:
done